### ISSUE TITLE
TypeError: undefined is not an object (evaluating 'targetEl.contentDocument')

### ISSUE THREAD

**Issue by diadal** [quasarframework/quasar#8590]
this issue only happened iOs Safari and some page form won't display correctly unless refreshing the page

```
Dev mode.......... spa
 Pkg quasar........ v2.0.0-beta.8
 Pkg @quasar/app... v3.0.0-beta.8
```
<img width="973" alt="Screen Shot 2021-03-08 at 5 33 17 PM" src="https://user-images.githubusercontent.com/19895516/110350997-9e742a80-8034-11eb-9880-81d3e337cf95.png">

**Comment by rstoenescu** [comment]
Please provide some context. A reproduction repo? Your details are not actionable.

**Comment by diadal** [comment]
check with this `https://next.quasar.dev/vue-components/bar` this issue come on refresh install too tested on different iPhone 6 7 11 same issue safari browser

```

<template>
  <div class="q-pa-md">
    <q-layout view="lHh lpr lFf" container style="height: 400px" class="shadow-2 rounded-borders">
      <q-header elevated>
        <q-bar>
          <q-icon name="laptop_chromebook" />
          <div>Google Chrome</div>

          <q-space />

          <q-btn dense flat icon="minimize" />
          <q-btn dense flat icon="crop_square" />
          <q-btn dense flat icon="close" />
        </q-bar>

        <div class="q-pa-sm q-pl-md row items-center">
          <div class="cursor-pointer non-selectable">
            File
            <q-menu>
              <q-list dense style="min-width: 100px">
                <q-item clickable v-close-popup>
                  <q-item-section>Open...</q-item-section>
                </q-item>
                <q-item clickable v-close-popup>
                  <q-item-section>New</q-item-section>
                </q-item>

                <q-separator />

                <q-item clickable>
                  <q-item-section>Preferences</q-item-section>
                  <q-item-section side>
                    <q-icon name="keyboard_arrow_right" />
                  </q-item-section>

                  <q-menu anchor="top end" self="top start">
                    <q-list>
                      <q-item
                        v-for="n in 3"
                        :key="n"
                        dense
                        clickable
                      >
                        <q-item-section>Submenu Label</q-item-section>
                        <q-item-section side>
                          <q-icon name="keyboard_arrow_right" />
                        </q-item-section>
                        <q-menu auto-close anchor="top end" self="top start">
                          <q-list>
                            <q-item
                              v-for="n in 3"
                              :key="n"
                              dense
                              clickable
                            >
                              <q-item-section>3rd level Label</q-item-section>
                            </q-item>
                          </q-list>
                        </q-menu>
                      </q-item>
                    </q-list>
                  </q-menu>
                </q-item>

                <q-separator />

                <q-item clickable v-close-popup>
                  <q-item-section>Quit</q-item-section>
                </q-item>
              </q-list>
            </q-menu>
          </div>

          <div class="q-ml-md cursor-pointer non-selectable">
            Edit
            <q-menu auto-close>
              <q-list dense style="min-width: 100px">
                <q-item clickable>
                  <q-item-section>Cut</q-item-section>
                </q-item>
                <q-item clickable>
                  <q-item-section>Copy</q-item-section>
                </q-item>
                <q-item clickable>
                  <q-item-section>Paste</q-item-section>
                </q-item>
                <q-separator />
                <q-item clickable>
                  <q-item-section>Select All</q-item-section>
                </q-item>
              </q-list>
            </q-menu>
          </div>
        </div>
      </q-header>

      <q-page-container>
        <q-page class="q-pa-md">
          <p v-for="n in 15" :key="n">
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit nihil praesentium molestias a adipisci, dolore vitae odit, quidem consequatur optio voluptates asperiores pariatur eos numquam rerum delectus commodi perferendis voluptate?
          </p>
        </q-page>
      </q-page-container>
    </q-layout>
  </div>
</template>
```

<img width="1009" alt="Screen Shot 2021-03-10 at 11 51 34 AM" src="https://user-images.githubusercontent.com/19895516/110618603-30944400-8197-11eb-9f22-e6fd109c55ed.png">
<img width="1006" alt="Screen Shot 2021-03-10 at 11 52 12 AM" src="https://user-images.githubusercontent.com/19895516/110618662-40ac2380-8197-11eb-9cae-8f0d8be00806.png">

**Comment by diadal** [comment]
is addition any idea i resolve this `Blocked a frame with origin "http://192.168.0.193:8086" from accessing a frame with origin "null".  The frame requesting access has a protocol of "http", the frame being accessed has a protocol of "about". Protocols must match.`

**Comment by fomalhaut79** [comment]
I had the same issue while using QResizeObserver with Quasar v2 Beta.

**Comment by rstoenescu** [comment]
Safari changing behaviour like this is really disturbing. This was a piece of code that worked flawlessly for years.
However, what iOS are we talking about here? Because iOS 13 should have a capability that doesn't require Quasar to use an iframe.

**Comment by diadal** [comment]
both Capacitor and SPA on iOS 13.3 & iOS 12 and SPA on MacBook catalina safari Version 14.0.2 (15610.3.7.1.10, 15610) 
the issue comes up

**Comment by rstoenescu** [comment]
Many thanks to @diadal for contributing with a PR.
Fix will be available in Quasar v2.0.0-beta.13.
